res/css/views/elements/_RichText.scss

```diff
@@ -80,6 +80,7 @@ a.mx_Pill {
     // and scale with the size of the surrounding text
     font-size: calc(18 / 14 * 1em);
     vertical-align: bottom;
+    font-family: Segoe UI Symbol;
```

Comment:
I'm not really concerned with having the emoji themselves be rainbow, and we really shouldn't switch from Twemoji to a font that is only available on Windows systems, so please remove this change.

Reply:
Okay, I have removed the change.